experimental/cmd/ccs811/main.go

```diff
@@ -18,31 +19,62 @@ import (
 	"periph.io/x/periph/host"
 )
 
+func justOneCommand(*flag.Flag) {
+
+}
+
 func main() {
+	i2cID := flag.String("i2c", "", "I²C bus to use (default, uses the first I²C found)")
+	i2cAddr := flag.Uint("ia", 0x5A, "I²C bus address to use; either 0x5A (90, default) or 0x5B (91)")
+	status := flag.Bool("status", false, "command displays status register of sensor")
+	rawData := flag.Bool("rawdata", false, "command displays current and voltage of sensors measurement resistor")
+	baseline := flag.Bool("baseline", false, "command displays value used for correction of measurement")
+	sense := flag.Bool("sense", false, "command performs one time measurement")
+	readContinuously := flag.Bool("readcontinuously", false, "command performs continous measuremnt with interval of one second")
+	fwInfo := flag.Bool("fwinfo", false, "command displays different versions of hardware, boot and firmware")
+	appStart := flag.Bool("appstart", false, "command starts sensor's application - move it from boot to application mode")
+	printMeasureMode := flag.Bool("printmeasuremode", false, "command shows current measurement mode")
+	setMeasureMode := flag.Uint("setmeasuremode", 1, "command sets mode, valid values are 0-4, default is periodic 1s reading")
+	flag.Parse()
+
+	numberOfCommandFlags := 0
```

Comment:
Oh I was fine to use flag.Args()[0] as the command, sorry I hadn't been clear.

Reply:
Well in this particular case, flag package sucks. I can't creat mutually exclusive commands (and I dont want to have -cmd <list of 10 possible parameters>), and to have same command in read and write mode just adds complexity. So I've split it.

Reply:
What I meant is to not specify them as flags, and instead look at the value of `flags.Args()[0]`. You can override the help page with `flag.Usage = func() {...}`.